gromacs/cmake/gmxManageColvars.cmake

```diff
@@ -38,6 +38,19 @@ gmx_option_multichoice(GMX_USE_COLVARS
     INTERNAL NONE)
 mark_as_advanced(GMX_USE_COLVARS)
 
+function(gmx_set_colvars_torch)
+  find_package(Torch)
```

Comment:
Note: This line will have to be removed for GMX 2025

Reply:
@HubLot I propose to stop patching any build recipe for the engines, and just test the features inside the library itself. I added support for linking against PyTorch in tests in PR #721, and tagged you as reviewer since you already reviewed a similar MR for GROMACS ;-)

Reply:
I agree!